daemon/util_wayland/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+project(util_wayland CXX)
+
+find_package(OpenSSL REQUIRED)
+pkg_check_modules(LIBDRM REQUIRED libdrm)
+
+include_directories( ../ )
+include_directories(${LIBDRM_INCLUDE_DIRS})
+include_directories( $ENV{WLD}/include $ENV{WLD}/include/libdrm  ../../external/libdrm ../../external/libdrm/include/drm wl_base )
+
+
+set (CMAKE_CXX_STANDARD 11)
+
+add_definitions(-g -Wall)
+
+add_library( ${PROJECT_NAME} SHARED
```

Comment:
is util_wayland a open source library, if so. it's better not include all the source code into hdcp. For example, hdcp depends on libdrm, but needn't include all the libdrm source code.

Reply:
No, util_wayland is not open source. I wrote it for another project and it fits nicely in this one so provided the source here too.

Reply:
@satyeshwar ok, it's fine to include these files for now, is there any plan for util_wayland open source?